diplom.py

```diff
@@ -19,7 +27,7 @@ def mean_average_precision(predicted_probability_distribution,
         classes_by_prob = []
         for i in xrange(0, len(list_of_classes)):
             classes_by_prob.append([row[i], list_of_classes[i]])
-            classes_by_prob.sort(reverse=True)    
+        classes_by_prob.sort(reverse=True)    
         for j in range (0, 5):
```

Comment:
Here is the output, thing I don't understand is why the precision of output is higher on log. regression? Random forest gives significantly better result.

Roman-computer:Diploma oksana$ ./diplom.py --model random_forest
Model used in this run is random_forest
MAP on test data: 0.14479
Map on train data: 0.72508

Roman-computer:Diploma oksana$ ./diplom.py --model logistic_regression
Model used in this run is logistic_regression
/Library/Python/2.7/site-packages/numpy-1.11.0-py2.7-macosx-10.10-intel.egg/numpy/core/fromnumeric.py:2652: VisibleDeprecationWarning: `rank` is deprecated; use the `ndim` attribute or function instead. To find the rank of a matrix see `numpy.linalg.matrix_rank`.
  VisibleDeprecationWarning)
MAP on test data: 0.0562411111111
Map on train data: 0.0691433333333